vuepress/docs/next/tutorials/devops/external-database/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This document provides a guide to connecting to an external database. In
+This document provides a guide to connecting an Entando Application to an external database. In
```

Comment:
Should these edits be applied to v6.3.0 as well?

Reply:
Yeah. I think the quickstart version is still accurate there but this is more comprehensive for most use cases.

Reply:
Done.